src/types/types.ts

```diff
@@ -274,3 +274,46 @@ export interface RevokeCollectionAuthorityRequest {
   revokeAuthorityKeypair: Keypair;
   payerKeypair?: Keypair;
 }
+
+// RWA Asset Types
+interface AssetControllerAccount {
+  address: string;
+  mint: string;
+  authority: string;
+  delegate: string;
+  version: number;
+  closed: boolean;
+}
+
+interface DataRegistryAccount {
+  address: string;
+  mint: string;
+  version: number;
+  closed: boolean;
+}
+
+interface IdentityRegistryAccount {
+  address: string;
+  mint: string;
+  authority: string;
+  delegate: string;
+  version: number;
+  closed: boolean;
+}
+
+interface PolicyEngine {
+  address: string;
+  mint: string;
+  authority: string;
+  delegate: string;
+  policies: string[];
+  version: number;
+  closed: boolean;
+}
+
+export interface FullAccount {
```

Comment:
```suggestion
export interface FullRwaAccount {
```
Just to avoid confusion